src/parser/parser.cc

```diff
@@ -605,30 +605,42 @@ class Parser {
     return ast;
   }
 
+  struct MetaRef {
```

Comment:
Are changes in this file relevant?

Reply:
Yes they fix a bug in the parsing which prevents this from working on all source files. I was going to turn on annotate spans for all models.